blocks/banner/banner.js

```diff
@@ -27,7 +25,7 @@ export default async function init(block) {
           const bannerImage = document.createElement('div');
           const bannerText = document.createElement('div');
           bannerImage.classList.add('banner-image');
-          bannerText.classList.add('banner-text');
+          bannerText.classList.add('banner-text', 'dark');
```

Comment:
Is the .dark option something that should be author-able or is this a new default? With out too much context on existing vs new, I just wonder if this will have any regression impact.

Reply:
I don't think we should bake this in. As Ryan P mentioned it should either use the default or be authorable. We do not want to break existing pages using the banner.

Reply:
Thanks for the feedback, added in variant for `light`, `dark`, `no-overlay` for more customization options
Staging link for `banner` block here: https://redesign-article--blog--adobecom.hlx.page/en/drafts/snowie-testing/blocks/banner